user/enterprise/operations-manual.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Travis CI Enterprise Operations manual
+layout: en_enterprise
+
+---
+
+This document is meant as a guideline to follow in case your Travis CI Enterprise production installation becomes unresponsive or unavailable.
+
+If this happens, we'd like to ask you to follow the steps outlined in this document first before you get in touch with us to save important data and collect as much evidence as possible. This will help us to provide you the best support possible and focus completely on resolving the problem.
+
+## First: Don't panic
+
+The first thing you should do is: keep calm. We're here to help and we're confident that we can bring it back online together with you!
+
+## Prerequisites
+
```

Comment:
This isnt a step, and also won't help when you get to this point. Fold it into the sentence on line 9 maybe, and probably shorten it a bit as well

Reply:
I agree with the rationale, but I think we should put this step after line 21 - it's about support, after all